script/upgrades/MU01/governance/Config.sol

```diff
@@ -354,4 +359,68 @@ library Config {
       config.minimumReports = 2;
     }
   }
+
+  function cEURUSDCConfig(Contracts.Cache storage contracts, uint8 phase) internal returns (PoolConfiguration memory config) {
+    require(phase >= 1 && phase <= 3, "phase must be 1, 2, or 3");
+    config = PoolConfiguration({
+      asset0: contracts.celoRegistry("StableTokenEUR"),
+      asset1: contracts.dependency("BridgedUSDC"),
+      isConstantSum: true,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      minimumReports: 5,
+      referenceRateResetFrequency: 5 minutes,
+      stablePoolResetSize: 1_800_000 * 1e18, // 12 million
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(2, 100), // 0.02
+      medianDeltaBreakerCooldown: 15 minutes,
+      smoothingFactor: FixidityLib.newFixedFraction(5, 10000), // 0.0005
+      isValueDeltaBreakerEnabled: true,
+      valueDeltaBreakerThreshold: FixidityLib.newFixedFraction(5, 1000), // 0.005
+      valueDeltaBreakerReferenceValue: 1e24, // 1$ numerator for 1e24 denominator
+      valueDeltaBreakerCooldown: 1 seconds,
+      referenceRateFeedID: contracts.dependency("USDCEURRateFeedAddr"),
+      asset0_timeStep0: 5 minutes,
+      asset0_timeStep1: 1 days,
+      asset0_limit0: [10_000, 10_000, 500_000][phase - 1],
+      asset0_limit1: [50_000, 50_000, 1_000_000][phase - 1],
+      asset0_limitGlobal: [50_000, 5_000_000, 14_000_000][phase - 1],
+      asset0_flags: L0 | L1 | LG
+    });
+
+    if (Chain.isBaklava() || Chain.isAlfajores()) {
+      config.minimumReports = 2;
+    }
+  }
+
+  function cBRLUSDCConfig(Contracts.Cache storage contracts, uint8 phase) internal returns (PoolConfiguration memory config) {
+    require(phase >= 1 && phase <= 3, "phase must be 1, 2, or 3");
+    config = PoolConfiguration({
+      asset0: contracts.celoRegistry("StableTokenBRL"),
+      asset1: contracts.dependency("BridgedUSDC"),
+      isConstantSum: true,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      minimumReports: 5,
+      referenceRateResetFrequency: 5 minutes,
+      stablePoolResetSize: 1_800_000 * 1e18, // 12 million
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(25, 1000), 0.025
+      medianDeltaBreakerCooldown: 15 minutes,
+      smoothingFactor: FixidityLib.newFixedFraction(5, 10000), //0.0005 
+      isValueDeltaBreakerEnabled: true,      
```

Comment:
@ninocomputer @bayological `cBRLUSDC` `valueDeltaBreaker` should be referencing `USDCUSD`. I don't know the name of  the parameter.

Reply:
did we decide that we should reference this in pool configs

Reply:
I am not sure. Just want to make sure it's not missed